relay-spans/src/lib.rs

```diff
@@ -0,0 +1,4 @@
+pub use crate::span::Span;
```

Comment:
Let's be pedantic about docs.

```suggestion
#![warn(missing_docs)]
#![doc(
    html_logo_url = "https://raw.githubusercontent.com/getsentry/relay/master/artwork/relay-icon.png",
    html_favicon_url = "https://raw.githubusercontent.com/getsentry/relay/master/artwork/relay-icon.png"
)]

pub use crate::span::Span;
```